### ISSUE TITLE
[sub-issue] CRw: Contrast insufficient in active or inactive elemen in navigation

### ISSUE THREAD

**Issue by bit-pro-iew-eui** [swiss/styleguide#321]
![image](https://cloud.githubusercontent.com/assets/7510842/5361736/ee724aee-7fcd-11e4-9f8b-dc9f2d05e745.png)

Contrast in active or inactive element in navigation is insufficient.

Please make it like this:
![image](https://cloud.githubusercontent.com/assets/7510842/5361830/9ebf07b6-7fce-11e4-9a8f-4d6f2bcd24f7.png)
==> Set the active menu point (Layouts) to <strong>bold</strong> and make the color darker #333333

**Comment by tonifisler** [comment]
Ugly code will be updated with #322.
